optaplanner-examples/src/main/resources/org/optaplanner/examples/travelingtournament/solver/travelingTournamentScoreRules.drl

```diff
@@ -35,19 +35,19 @@ global HardSoftScoreHolder scoreHolder;
 rule "fourConsecutiveHomeMatches"
     when
         Match($homeTeam : homeTeam, $firstDayIndex : day.getIndex())
-        Match(homeTeam == $homeTeam, day.getIndex() == $firstDayIndex + 1)
-        Match(homeTeam == $homeTeam, day.getIndex() == $firstDayIndex + 2)
-        Match(homeTeam == $homeTeam, day.getIndex() == $firstDayIndex + 3)
+        exists Match(homeTeam == $homeTeam, day.getIndex() == $firstDayIndex + 1)
+        exists Match(homeTeam == $homeTeam, day.getIndex() == $firstDayIndex + 2)
+        exists Match(homeTeam == $homeTeam, day.getIndex() == $firstDayIndex + 3)
     then
         scoreHolder.addHardConstraintMatch(kcontext, -1);
 end
 
 rule "fourConsecutiveAwayMatches"
     when
         Match($awayTeam : awayTeam, $firstDayIndex : day.getIndex())
-        Match(awayTeam == $awayTeam, day.getIndex() == $firstDayIndex + 1)
-        Match(awayTeam == $awayTeam, day.getIndex() == $firstDayIndex + 2)
-        Match(awayTeam == $awayTeam, day.getIndex() == $firstDayIndex + 3)
+        exists Match(awayTeam == $awayTeam, day.getIndex() == $firstDayIndex + 1)
+        exists Match(awayTeam == $awayTeam, day.getIndex() == $firstDayIndex + 2)
+        exists Match(awayTeam == $awayTeam, day.getIndex() == $firstDayIndex + 3)
```

Comment:
Does this change have any impact on the DRL performance?

Reply:
I actually did that to make sure that CS-D and DRL are really 1:1.
It did improve the DRL performance, but CS-D is still better. This is curious.